apps/aehttp/src/swagger/swagger_api.erl

```diff
@@ -0,0 +1,325 @@
+-module(swagger_api).
```

Comment:
Shouldn't these modules follow the 'ae' prefix naming conventions?
Presumably, `aeh_` would be a suitable prefix.

Reply:
These are swagger generated files: of course we can rename them, but we will have to rename them for every new swagger code generation. If there is a way to config swagger to follow our naming convention - I'm not aware of it.

Reply:
It seems as if you can do it with the option `PACKAGE_NAME`
https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ErlangServerCodegen.java#L38

Presumably, this can be set in a config file, using the `--config` option, but looking around, this wasn't very well documented, and the swagger_codegen help text (at least my version) didn't even know what erlang was. This may well have been due to some mistake on my part.